starport/interface/cli/starport/cmd/module-create.go

```diff
@@ -17,13 +46,41 @@ func NewModuleCreate() *cobra.Command {
 		Args:  cobra.MinimumNArgs(1),
 		RunE:  createModuleHandler,
 	}
+	c.Flags().Bool(flagIBC, false, "scaffold an IBC module")
+	c.Flags().String(flagIBCOrdering, "none", "channel ordering of the IBC module [none|ordered|unordered]")
 	return c
 }
 
 func createModuleHandler(cmd *cobra.Command, args []string) error {
+	var options []scaffolder.ModuleCreationOption
+
+	// Check if the module must be an IBC module
+	ibcModule, err := cmd.Flags().GetBool(flagIBC)
+	if err != nil {
+		return err
+	}
+
+	if ibcModule {
+		options = append(options, scaffolder.WithIBC())
```

Comment:
We can merge this inside the one below > https://github.com/tendermint/starport/pull/731/files#diff-e5901da6a8b97970975f24ac956d36b23b29a224a6dcbad31e1605557ace7992R71